apps/web/src/pages/get-started/consts/EchoUseCase.const.tsx

```diff
@@ -0,0 +1,110 @@
+import { StepDescription, StepText } from './shared';
+import { OnboardingUseCase } from './types';
+import { OnboardingUseCasesTabsEnum } from './OnboardingUseCasesTabsEnum';
+import { colors, Copy } from '@novu/design-system';
+import { Group, UnstyledButton, useMantineColorScheme } from '@mantine/core';
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Please avoid direct imports to `@mantine/core` as discussed at the R&D offsite‏